utils/scripts/update_dashboard_CI_visibility.sh

```diff
@@ -0,0 +1,39 @@
+if [ "$#" -ne 2 ] ; then
+  echo "Script execution requires 2 parameters: Pipeline id and dashboard variable name" 
+  exit 1
+fi
+SYS_PIPELINE_RUN_ID=$1
+DASHBOARD_VARIABLE_NAME=$2
+
+echo "PIPELINE RUN ID $SYS_PIPELINE_RUN_ID"
+
+# Path parameters
+export dashboard_id="zqg-kqn-2mc"
+
+export dashboard_json=$(
+curl -X GET "https://api.datadoghq.com/api/v1/dashboard/${dashboard_id}" \
+-H "Accept: application/json" \
+-H "DD-API-KEY: ${DD_API_KEY}" \
+-H "DD-APPLICATION-KEY: ${DD_APP_KEY}")
+
+#Search in all template variables (no more than 4) trying to find DASHBOARD_VARIABLE_NAME pass as parameter
+#When I achieve to find the variable, I will change the default value with the last pipelineId + run attemp number 
+for counter in 0 1 2 3  
+do
+    dashboard_json_variable=$(jq -r ".template_variables[$counter].prefix"  <<< "$dashboard_json")
```

Comment:
maybe worthy to have a look on jq `select` feature

Reply:
We will probably change the number of dashboard variables. This script is not final version